tests/logictest/http_connector.py

```diff
@@ -52,8 +52,7 @@ def format_result(results):
             if buf == "":
                 buf = str(item)
             else:
-                buf = buf + " " + str(
-                    item)  # every item seperate by space
+                buf = buf + " " + str(item)  # every item seperate by space
```

Comment:
this unrelated pr can be rollback?

Reply:
this is changed by `make lint`, I'll roll it back.